README.md

```diff
@@ -56,6 +56,22 @@ make serve
 
 This will start the local Hugo server on port 1313. Open up your browser to http://localhost:1313 to view the website. As you make changes to the source files, Hugo updates the website and forces a browser refresh.
 
+## Rebuilding the API reference pages
+
+The API reference pages located in `content/en/docs/reference/kubernetes-api/ref` are built from the Swagger specifications, using https://github.com/feloy/kubernetes-resources-reference.
```

Comment:
nit: Now is a good time to look at the file hierarchy of the reference section before new content is introduced.
nit: I think the left hand nav will need to adapt (or change) to display more content. I have a hard time navigating the content as it is.
nit: I don't know whether `ref` is the right choice for the root `reference` directory name.

Reply:
@kbhawkey 

> I think the left hand nav will need to adapt (or change) to display more content. I have a hard time navigating the content as it is.

Agreed, I offered own feedback to this effect. 

> nit: Now is a good time to look at the file hierarchy of the reference section before new content is introduced.

Agreed. I think file pathing is more important than a nit, and I also think it's an easy iteration to make post-merge.

Reply:
Yes, it is important to get the supporting file structure right from the start.
At least get the root directories right before putting this new file tree in place.